### DISCUSSION TITLE
Is *.otf file supported?

### DISCUSSION THREAD

**Discussion by Ninja2790t** [SixLabors/ImageSharp#1648] [Q&A] [answered]
```
var fontCollection = new FontCollection();
fontCollection.Install(new MemoryStream(otfFileByteArray));

```
This is throwing exception "invalid glyph format, only ttf glyph outlines supported. imagesharp".

**Comment by tocsoft** [comment, marked as answer]
*.otf files will work fine as longs as they contain true type outlines...

*.ttf fonts are actually a subset of *.otf files but specifically with true type glyph outlines.
